src/cli/commands/test/cache.rs

```diff
@@ -1,17 +1,27 @@
 pub mod cache {
+	use dirs;
 	use std::{fmt::Debug, fs::read_to_string, io, path::PathBuf};
 
 	use thiserror::Error;
 
 	use serde::{Deserialize, Serialize};
 	use serde_json;
 
+	const CAIRO_FOUNDRY_CACHE_DIR: &str = "cairo-foundry-cache";
+	const CAIRO_FOUNDRY_COMPILED_CONTRACT_DIR: &str = "compiled-cairo-files";
+
 	#[derive(Error, Debug)]
 	pub enum CacheError {
 		#[error(transparent)]
 		FileNotFoundError(#[from] io::Error),
 		#[error(transparent)]
 		DeserializeError(#[from] serde_json::Error),
+		#[error("cache directory does not exist on this platform")]
+		CacheDirNotSupportedError,
+		#[error("filename does not exist")]
```

Comment:
is this the correct error message? `InvalidContractExtension` sounds like the file has a wrong extension?